app/js/HttpController.js

```diff
@@ -179,8 +179,9 @@ module.exports = HttpController = {
     }
     const { project_id } = req.params
     const { doc_id } = req.params
+    const { name } = req.query
```

Comment:
It looks like we're not validating that `name` exists anywhere. Should we? -- I don't think we want to be deleting stuff without this data now. - I see no harm in adding this to the log line below, either.

Reply:
Good point, done with joi in super seeding PR https://github.com/overleaf/docstore/pull/93